tracer/test/Datadog.Trace.TestHelpers/TestHelper.cs

```diff
@@ -24,8 +29,10 @@
 
 namespace Datadog.Trace.TestHelpers
 {
-    public abstract class TestHelper
+    public abstract class TestHelper : IDisposable
     {
+        private bool _deleteRcmFile = true;
```

Comment:
I think we _always_ want to delete this file, except of manual tests debug, so not sure we need this flag.

Reply:
True. I'll change it

Reply:
Done in c5cf6dc713be9d83186bc89361d8c9524215252e